README.md

```diff
@@ -69,32 +69,21 @@ print('Setting value to: Nihao')
 print('Contract value: {}'.format(contract_instance.greet()))
 ```
 
-## Developer setup
-
-If you would like to hack on web3.py, set up your dev environment with:
+## Developer Setup
 
-```sh
-sudo apt-get install libssl-dev libffi-dev autoconf automake libtool
-# ^ This is for Debian-like systems. TODO: Add more platforms
+Please see OS-specific instructions for:
 
-sudo pacman -Sy libsecp256k1
-# ^ This is for ArchLinux system
+- [Linux](docs/README-linux.md#Developer-Setup)
+- [Mac](docs/README-osx.md#Developer-Setup)
+- [Windows](docs/README-windows.md#Developer-Setup)
 
-sudo dnf install openssl-devel libffi-devel autoconf automake libtool
-# ^ This is for Fedora.
+Then run these install commands:
 
-git clone git@github.com:ethereum/web3.py.git
-cd web3.py
-virtualenv venv
-. venv/bin/activate
-pip install -r requirements-dev.txt
-pip install -e .[tester]
+```
+pip install -e . -r requirements-dev.txt
```

Comment:
This dropped the testing extra. Should be: `pip install -e .[tester] -r requirements-dev.txt`

Also, this must be run after switching to the relevant virtualenv, so the venv setup instructions should come first.